src/main/java/appeng/crafting/CraftingJob.java

```diff
@@ -47,20 +47,20 @@
 public class CraftingJob implements Runnable, ICraftingJob
 {
 
-	final IItemList<IAEItemStack> storage;
-	final Set<IAEItemStack> prophecies;
-	final MECraftingInventory original;
-	final World world;
-	final IItemList<IAEItemStack> crafting = AEApi.instance().storage().createItemList();
-	final IItemList<IAEItemStack> missing = AEApi.instance().storage().createItemList();
-	final HashMap<String, TwoIntegers> opsAndMultiplier = new HashMap<String, TwoIntegers>();
+	private final IItemList<IAEItemStack> storage;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "storage" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1068)